grc/core/utils/extract_docs.py

```diff
@@ -277,7 +275,7 @@ def send(code, cmd, args):
 elif __name__ == '__main__':
     def callback(key, docs):
         print(key)
-        for match, doc in six.iteritems(docs):
+        for match, doc in docs.itmes():
```

Comment:
This is a typo. Did you test all of these changes?

Reply:
Sorry, will check again everything for typos, I've checked most of the changes.